sql-plugin/src/main/scala/com/nvidia/spark/rapids/CostBasedOptimizer.scala

```diff
@@ -264,11 +297,7 @@ class DefaultCostModel(conf: RapidsConf) extends CostModel {
         case _ => conf.defaultOperatorCost
       }
     }
-
-    plan.cpuCost = cpuCost
-    plan.gpuCost = gpuCost
-
-    (cpuCost, gpuCost)
+    rowCount * perRowCost
```

Comment:
This is a good first step but longer term I think we might want to turn this model on its head a bit and pass the row count to the cost estimation for the expression.  On the GPU there is a fixed cost to launching a kernel, so `5 + foo` has some cost that is fixed no matter the number of rows, and a second potion that is relative to the cost of doing the add. If we don't include this in some form then we cannot fall back to the CPU for really small row counts.